plans/unittests.fmf

```diff
@@ -2,10 +2,8 @@ summary:
     Unit tests
 execute:
     script:
-    - pwd; cd ~/git-source && pwd && nosetests -v
+    - pwd; cd ~/git-source && pwd && ./run_unit_tests.sh
 prepare:
     how: shell
     script:
-    - yum --enablerepo=extras install -y epel-release
-    - yum install -y python-pip
-    - pip install nose
+    - yum install -y python-nose
```

Comment:
@thrix, I want to run the unit tests with epel 6, 7, and 8. But with epel 8, there's `python3-nose` available, instead of `python-nose`. 

Do you suggest to create two fmf files, each using `relevancy`? E.g.
```
relevancy:
- "distro = rhel-6, rhel-7: True"
```
Or to rather use a shell condition? E.g.:
```
if [ command -v dnf ]; then dnf install python3-nose; else yum install -y python-nose; fi
```

Reply:
Ah, the testing farm run `ln -fs $(which yum) /usr/bin/dnf` so I can't use the `command -v dnf` test.

Reply:
Ok, this doesn't work as well: `if command -v python3 &> /dev/null ; then dnf install python3-nose; else yum install -y python-nose; fi`

Reply:
So, the `relevancy` is being ignored completely. I'm out of ideas.

Reply:
Related RFE: https://github.com/psss/tmt/issues/101